src/bin/cargo-fmt.rs

```diff
@@ -288,16 +288,16 @@ fn format_files(files: &[PathBuf],
         println!("");
     }
     let mut command = try!(Command::new("rustfmt")
-        .stdout(stdout)
-        .args(files)
-        .args(fmt_args)
-        .spawn()
-        .map_err(|e| match e.kind() {
-            std::io::ErrorKind::NotFound => {
-                std::io::Error::new(std::io::ErrorKind::Other,
-                                    "Could not run rustfmt, please make sure it is in your PATH.")
-            }
-            _ => e,
-        }));
+                               .stdout(stdout)
```

Comment:
true, although it is a shame to break formatting which was working well before (I guess that if it is not possible to fix the bug here without this bustage, then it is OK)